README.md

```diff
@@ -42,17 +47,21 @@ the supported version and an IDE of your choice installed.
 It is also recommended to sign up for the Camunda Cloud trial in front of the workshop. This trial will last for 30 days, 
 so make sure it is still active when the workshop is taking place. You can sign up [here](https://accounts.cloud.camunda.io/signup).
 
-# 🎓Excercises
-## Excercise 2: Get familiar with Camunda Cloud and discover features 
+# 🎓Exercises
+
+## Exercise 1: Design the process
+
+You can design your own process using using the [Camunda Modeler](https://camunda.com/download/modeler/), or the Web Modeler in Camunda Cloud. 
```

Comment:
🔧 - Could you provide a link to the web modeler? I think we should use this one as the default tool for this workshop.

💭 - Maybe it is worth to quickly explain how to create a project + bpmn model in there.

Reply:
Yes, I can add this link.